<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="papeldeparede">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius: 10px;
	background-color: rgb(211, 196, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iconsenha">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/imgs/006-cadeado.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iconuser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/imgs/001-do-utilizador.png);
image: url(:/imgs/001-do-utilizador.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="esqueceusenha">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Esqueceu Senha?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEntrar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Entrar no Sistema&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(103, 103, 103);
	border-radius: 5px;
	border: 2px solid rgb(93, 93, 93);
}
QPushButton:hover{
	background-color: rgb(123, 123, 123);
	border-radius: 5px;
	border: 2px solid rgb(103, 103, 103);
}
QPushButton:pressed{
	background-color: rgb(23, 23, 23);
	border-radius: 5px;
	border: 2px solid rgb(33, 33, 33);
}</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icons/login.qrc">:/newPrefix/mascote.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSenha">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Digite senha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-radius: 5px;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsuario">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Digite login&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Digite login&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-radius: 5px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Digite Usuário</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSair">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Fechar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(103, 103, 103);
	border-radius: 5px;
	border: 2px solid rgb(93, 93, 93);
}
QPushButton:hover{
	background-color: rgb(123, 123, 123);
	border-radius: 5px;
	border: 2px solid rgb(103, 103, 103);
}
QPushButton:pressed{
	background-color: rgb(23, 23, 23);
	border-radius: 5px;
	border: 2px solid rgb(33, 33, 33);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cadastrar Usuário&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(103, 103, 103);
	border-radius: 5px;
	border: 2px solid rgb(93, 93, 93);
	image: url(:/imgs/001-do-utilizador.png);
}
QPushButton:hover{
	background-color: rgb(123, 123, 123);
	border-radius: 5px;
	border: 2px solid rgb(103, 103, 103);
}
QPushButton:pressed{
	background-color: rgb(23, 23, 23);
	border-radius: 5px;
	border: 2px solid rgb(43, 43, 43);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errou">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(255, 255, 255);
	border-radius: 45px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>papeldeparede</zorder>
   <zorder>label</zorder>
   <zorder>iconsenha</zorder>
   <zorder>iconuser</zorder>
   <zorder>esqueceusenha</zorder>
   <zorder>btnEntrar</zorder>
   <zorder>titulo</zorder>
   <zorder>leSenha</zorder>
   <zorder>leUsuario</zorder>
   <zorder>btnSair</zorder>
   <zorder>btnCadastrar</zorder>
   <zorder>errou</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="actionSistema">
   <property name="text">
    <string>Sistema</string>
   </property>
  </action>
  <action name="actionVAmos">
   <property name="text">
    <string>VAmos</string>
   </property>
  </action>
  <action name="actionLe">
   <property name="text">
    <string>Le</string>
   </property>
  </action>
  <action name="actionSD">
   <property name="text">
    <string>SD</string>
   </property>
  </action>
  <action name="actionSJDKJWI">
   <property name="text">
    <string>SJDKJWI</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/login.qrc"/>
  <include location="../icons/principal.qrc"/>
 </resources>
 <connections/>
</ui>
